docs/05-technical-reference/00-custom-resources/evnt-02-eventingbackend.md

```diff
@@ -0,0 +1,41 @@
+---
+title: EventingBackend
+---
+
+The EventingBackend custom resource definition (CRD) is used to show the current status of the Eventing backend. To get the up-to-date CRD and see the output in the YAML format, run this command:
+
+```shell
+kubectl get crd eventingbackends.eventing.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample custom resource (CR) that the Eventing Controller creates by default when Kyma is deployed. It has an empty `spec` section.
+
+```yaml
+apiVersion: eventing.kyma-project.io/v1alpha1
+kind: EventingBackend
+metadata:
+  name: eventing-backend
+  namespace: kyma-system
+spec: {}
+```
```

Comment:
As we do not support more than one eventing backend and we also create this as soon as the eventing controller starts i think we do not need this section.

Reply:
Instead should I paste the output of the previous command:
```
➜  ~ kubectl get eventingbackends.eventing.kyma-project.io eventing-backend -n kyma-system                                                                                                                  (shoot--kymatunas--rp-ir/default)
NAME               BACKEND   EVENTINGREADY   SUBSCRIPTIONCONTROLLERREADY   PUBLISHERPROXYREADY
eventing-backend   NATS      true            True                          True
```

to show the actual usecase of this CR